src/BenchmarkTools.jl

```diff
@@ -12,7 +12,7 @@ using Profile
 const BENCHMARKTOOLS_VERSION = if VERSION >= v"1.9"
     pkgversion(BenchmarkTools)
 else
-    v"1.4.0"
+    v"1.6.0"
```

Comment:
Let's replace these 5 lines with:

```
const BENCHMARKTOOLS_VERSION = VersionNumber(TOML.parsefile(joinpath(dirname(@__DIR__), "Project.toml"))["version"])
```

Reply:
Perhaps we should use Compat.jl instead? There seems to be some sort of locking going on in `pkgversion` (https://github.com/JuliaLang/Compat.jl/blob/220f4d7d34e9de9c14fece4b43425565713f2b25/src/Compat.jl#L442) and I don't understand it well enough to know if your change would be fine.

Reply:
that would be fine as well! let's do that.